src/styles/blocks/stars--1.css

```diff
@@ -0,0 +1,9 @@
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
Consider using a general selector for `.stars__star` instead of repeating the same styles in multiple files (stars--1.css, stars--2.css, etc.). You can define the active star styles once and then only change the `nth-child` range in the modifier classes.